DepositeCalcTests/Tests/CalculatorPageTests.cs

```diff
@@ -0,0 +1,230 @@
+﻿using DepositeCalcTests.Pages;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+using System.Globalization;
+using System.Text.RegularExpressions;
+
+namespace DepositeCalcTests.Tests
+{
+    internal class CalculatorPageTests
+    {
+        private IWebDriver driver;
+
+        [SetUp]
+        public void Setup()
+        {
+            ChromeOptions options = new ChromeOptions { AcceptInsecureCertificates = true };
+            driver = new ChromeDriver(options);
+            driver.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(30);
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(5);
+            driver.Url = "https://localhost:5001/Calculator";
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            driver.Quit();
+        }
+
+        [TestCase("", "1", "1")]
+        [TestCase("1", "", "1")]
+        [TestCase("1", "1", "")]
+        public void MandaroryTextFieldsWith365BtnTest(string depositAmount, string interestRate, string investmentTerm)
```

Comment:
Now you also can merge this and the next tests into one, like this:
```suggestion
        [TestCase("360", "1", "1", "")]
        [TestCase("365", "1", "1", "")]
        public void MandaroryTextFieldsWith365BtnTest(string finYear, string depositAmount, string interestRate, string investmentTerm)
```